worker/transfer.go

```diff
@@ -31,10 +32,11 @@ var (
 
 // A sectorStore stores contract data.
 type sectorStore interface {
+	Account() rhpv3.Account
```

Comment:
I'd probably drop this if not every type that implements `sectorStore` can provide it and do a safe cast instead when you need access.
Either that or return `rhpv3.Account, bool`

Reply:
Decided to remove it, I only needed it for a span attribute anyway.